src/tree-internal.component.ts

```diff
@@ -108,6 +116,18 @@ export class TreeInternalComponent implements OnInit {
     this.treeService.fireNodeMoved(addedSibling, e.captured.tree.parent);
   }
 
+  public getTreeAPI(): TreeInternalAPI {
+    return this.api;
+  }
+
+  public getChildAPIById(id: number | string): TreeInternalAPI {
```

Comment:
I've removed the get API methods from tree-internal.component.ts because there is no need to have such methods, for now :)